### ISSUE TITLE
How to set XLA random seed

### ISSUE THREAD

**Issue by myleott** [pytorch/xla#2092]
## ❓ Questions and Help

On CPU (same from run to run):
```python
torch.manual_seed(0)
torch.zeros(5).uniform_()
# tensor([0.4963, 0.7682, 0.0885, 0.1320, 0.3074])

torch.manual_seed(0)
torch.zeros(5).uniform_()
# tensor([0.4963, 0.7682, 0.0885, 0.1320, 0.3074])
```

On XLA (different from run to run):
```python
torch.manual_seed(0)
torch.zeros(5, device=xm.xla_device()).uniform_()
# tensor([0.9650, 0.4818, 0.2164, 0.2308, 0.8543], device='xla:1')

torch.manual_seed(0)
torch.zeros(5, device=xm.xla_device()).uniform_()
# tensor([0.3197, 0.6271, 0.0868, 0.2445, 0.3315], device='xla:1')
```

**Comment by JackCaoG** [comment]
I don't think we have the python api to do this yet. @ailzhang was looking into this couple weeks ago, she might have a better idea. From CPP side, we have https://github.com/pytorch/xla/blob/master/torch_xla/csrc/tensor.cpp#L319 api to update the seed.

**Comment by myleott** [comment]
Nevermind, found it: `xm.set_rng_seed`

**Comment by dlibenzi** [comment]
https://github.com/pytorch/xla/blob/f598f9b88b27299304bff029e774c47ea7611b3f/torch_xla/core/xla_model.py#L716

I think @ailzhang was looking into hooking into pytorch one...

**Comment by myleott** [comment]
Yeah, actually just `set_rng_seed` won't support forking. Ideally I'd like to save the RNG state and restore it.
